### ISSUE TITLE
Trigger parameter panel information loss after setting the viewport to 320*256 pixels

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#7641]
### Severity

P2 - High (Major barrier to access)

### Type of Accessibility Issue

Other

### Describe the issue

Cannot view the information on trigger parameter panel when settings viewport to 320*256 pixiel or zoom to 400%

zoom 400%

![Image](https://github.com/user-attachments/assets/f451a2f8-fbb8-434a-99bf-00bc7a7e90cf)

320*256

![Image](https://github.com/user-attachments/assets/ade1eac3-2c51-4ce0-9a21-49c00a559343)

### What type of Logic App are you using?

Consumption (Portal)

### Environment

Browser: Edge

### WCAG Success Criteria (if known)

MAS 1.4.10 – Reflow
